nix_test.go

```diff
@@ -115,6 +115,29 @@ func TestProject(t *testing.T) {
 	runSubtestsWithGenerator(t, g)
 }
 
+func TestEnableOption(t *testing.T) {
```

Comment:
Can you add a copy of this test named `TestEnableOption_WithHeader` where we set `WriteHeader` to `true`?